src/jemalloc.c

```diff
@@ -1173,6 +1174,16 @@ malloc_conf_init(void) {
 				CONF_HANDLE_BOOL(opt_prof_final, "prof_final")
 				CONF_HANDLE_BOOL(opt_prof_leak, "prof_leak")
 			}
+			if (config_log) {
+				if (CONF_MATCH("log_vars")) {
+					size_t cpylen = (
+					    vlen <= sizeof(log_var_names) ?
+					    vlen : sizeof(log_var_names) - 1);
+					strncpy(log_var_names, v, cpylen);
+					log_var_names[cpylen] = '\0';
```

Comment:
Merging is a little hard just because of C string irritation stuff. For a purely-debugging thing it feels like it can be deferred until we need it?

I guess log_vars is always valid. Like, if someone says "turn on any logging for log vars named asgs$&^@(.\t\v\a" then it won't match any of the logvars we set, but I don't see the harm in letting them.